peachjam/static/stylesheets/_global.scss

```diff
@@ -151,6 +151,10 @@ footer {
   font-size: 22px;
 }
 
+.text-purple-500 {
```

Comment:
Minor: I suggest making this semantic (eg `.text-ai`) and not the exact colour, and applying the same colour for the `pj-ai` icon, so that you can change that colour globally (eg. for ULII).

Reply:
agreed, changed